tools/release_do_nothing.py

```diff
@@ -921,46 +1216,72 @@ def merge_back(self):
 
             message = (
                 "Commit and push all the What's New changes.\n"
-                f"git add {self.whats_news.index.absolute()};\n"
-                'git commit -m "Restore latest Whats New files.";\n'
-                f"git push -u origin {working_branch};"
+                f"git add {self.whats_news.index_.absolute()};\n"
+                'git commit -m "Restore latest Whats-New files.";\n'
+                f"git push -u {self.github_fork} {working_branch};"
             )
             self.wait_for_done(message)
 
-            self._create_pr(
-                base_org="SciTools",
-                base_repo="iris",
-                base_branch="main",
-                head_branch=working_branch,
-            )
-            message = (
-                "Work with the development team to get the PR merged.\n"
-                "Make sure the documentation is previewed during this process.\n"
-                f"{merge_commit}"
+        self._create_pr(
```

Comment:
Double-check whether we've got an instruction to push. We also need that instruction for the other PRs too